sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+sonar.projectKey=fh-logger
+sonar.projectName=fh-logger-nightly-master
+sonar.projectVersion=0.4.1
+
+sonar.sources=./lib
+sonar.tests=./test
+sonar.language=js
+sonar.javascript.lcov.reportPath=./coverage/lcov.info
```

Comment:
Sonnar cannot find this file and places comment. We would need to make sure that unit tests and coverage file is regenerated.